workspace.spec

```diff
@@ -572,6 +572,359 @@ module Workspace {
 		string description;
 	} ProvenanceAction;
 
+	/* Credit Metadata-related objects and fields
```

Comment:
where's the best place to put this info?

Reply:
IIRC we said we were going to expose it in https://ci.kbase.us/services/ws/docs/Workspace.html#typedefWorkspace.ObjectData 
Assuming I'm right, this seems like a good a place as any. If I'm wrong, and it's going in `ProvenanceAction`, then the de facto standard is to put the data structure definition prior to the use

Reply:
Note that if we don't include it as a part of the provenance data structure, we'll need to do a separate size check (and have a separate, hopefully very small, size limit)

Reply:
just to check: if it goes under ObjectData, will there be a single version for all copies and copies of copies (of copies of copies, etc.) that get made of the object?

Reply:
You could implement it either way, depending on what you wanted. The database schema and API schemas don't really have much to do with each other

Reply:
And FWIW we could keep it in the internal Provenance data structure internally and use the existing size check

Reply:
I've left the data structure where it is as the plan is to expose it as part of the workspace object data.